index.html

```diff
@@ -1,11 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Crystal Supermarket</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="header">
+      <nav>
+            <img src="img/Crystal_Supermarket_logo.png" alt="logo" />
+                <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#more">More</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+      </nav>
+    </div>
+
+    <section>
+
+      <div class="hero_heading">
+            <h1>Welcome Here</h1>
```

Comment:
Since this is just a single element, no need to put it in a div unless you will be adding some other element in the future.